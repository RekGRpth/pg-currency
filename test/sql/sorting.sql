SELECT * FROM (
  VALUES
  ('ZWD'::currency),
  ('ZMW'::currency),
  ('ZAR'::currency),
  ('YER'::currency),
  ('XPF'::currency),
  ('XOF'::currency),
  ('XDR'::currency),
  ('XCD'::currency),
  ('XAF'::currency),
  ('WST'::currency),
  ('VUV'::currency),
  ('VND'::currency),
  ('VEF'::currency),
  ('UZS'::currency),
  ('UYU'::currency),
  ('USD'::currency),
  ('UGX'::currency),
  ('UAH'::currency),
  ('TZS'::currency),
  ('TWD'::currency),
  ('TVD'::currency),
  ('TTD'::currency),
  ('TRY'::currency),
  ('TOP'::currency),
  ('TND'::currency),
  ('TMT'::currency),
  ('TJS'::currency),
  ('THB'::currency),
  ('SZL'::currency),
  ('SYP'::currency),
  ('SVC'::currency),
  ('STD'::currency),
  ('SRD'::currency),
  ('SPL'::currency),
  ('SOS'::currency),
  ('SLL'::currency),
  ('SHP'::currency),
  ('SGD'::currency),
  ('SEK'::currency),
  ('SDG'::currency),
  ('SCR'::currency),
  ('SBD'::currency),
  ('SAR'::currency),
  ('RWF'::currency),
  ('RUB'::currency),
  ('RSD'::currency),
  ('RON'::currency),
  ('QAR'::currency),
  ('PYG'::currency),
  ('PLN'::currency),
  ('PKR'::currency),
  ('PHP'::currency),
  ('PGK'::currency),
  ('PEN'::currency),
  ('PAB'::currency),
  ('OMR'::currency),
  ('NZD'::currency),
  ('NPR'::currency),
  ('NOK'::currency),
  ('NIO'::currency),
  ('NGN'::currency),
  ('NAD'::currency),
  ('MZN'::currency),
  ('MYR'::currency),
  ('MXN'::currency),
  ('MWK'::currency),
  ('MVR'::currency),
  ('MUR'::currency),
  ('MRO'::currency),
  ('MOP'::currency),
  ('MNT'::currency),
  ('MMK'::currency),
  ('MKD'::currency),
  ('MGA'::currency),
  ('MDL'::currency),
  ('MAD'::currency),
  ('LYD'::currency),
  ('LTL'::currency),
  ('LSL'::currency),
  ('LRD'::currency),
  ('LKR'::currency),
  ('LBP'::currency),
  ('LAK'::currency),
  ('KZT'::currency),
  ('KYD'::currency),
  ('KWD'::currency),
  ('KRW'::currency),
  ('KPW'::currency),
  ('KMF'::currency),
  ('KHR'::currency),
  ('KGS'::currency),
  ('KES'::currency),
  ('JPY'::currency),
  ('JOD'::currency),
  ('JMD'::currency),
  ('JEP'::currency),
  ('ISK'::currency),
  ('IRR'::currency),
  ('IQD'::currency),
  ('INR'::currency),
  ('IMP'::currency),
  ('ILS'::currency),
  ('IDR'::currency),
  ('HUF'::currency),
  ('HTG'::currency),
  ('HRK'::currency),
  ('HNL'::currency),
  ('HKD'::currency),
  ('GYD'::currency),
  ('GTQ'::currency),
  ('GNF'::currency),
  ('GMD'::currency),
  ('GIP'::currency),
  ('GHS'::currency),
  ('GGP'::currency),
  ('GEL'::currency),
  ('GBP'::currency),
  ('FKP'::currency),
  ('FJD'::currency),
  ('EUR'::currency),
  ('ETB'::currency),
  ('ERN'::currency),
  ('EGP'::currency),
  ('DZD'::currency),
  ('DOP'::currency),
  ('DKK'::currency),
  ('DJF'::currency),
  ('CZK'::currency),
  ('CVE'::currency),
  ('CUP'::currency),
  ('CUC'::currency),
  ('CRC'::currency),
  ('COP'::currency),
  ('CNY'::currency),
  ('CLP'::currency),
  ('CHF'::currency),
  ('CDF'::currency),
  ('CAD'::currency),
  ('BZD'::currency),
  ('BYR'::currency),
  ('BYN'::currency),
  ('BWP'::currency),
  ('BTN'::currency),
  ('BSD'::currency),
  ('BRL'::currency),
  ('BOB'::currency),
  ('BND'::currency),
  ('BMD'::currency),
  ('BIF'::currency),
  ('BHD'::currency),
  ('BGN'::currency),
  ('BDT'::currency),
  ('BBD'::currency),
  ('BAM'::currency),
  ('AZN'::currency),
  ('AWG'::currency),
  ('AUD'::currency),
  ('ARS'::currency),
  ('AOA'::currency),
  ('ANG'::currency),
  ('AMD'::currency),
  ('ALL'::currency),
  ('AFN'::currency),
  ('AED'::currency)
) t(c)
ORDER BY c;
