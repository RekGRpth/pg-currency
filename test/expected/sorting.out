SELECT * FROM (
  VALUES
  ('ZWD'::currency),
  ('ZMW'::currency),
  ('ZAR'::currency),
  ('YER'::currency),
  ('XPF'::currency),
  ('XOF'::currency),
  ('XDR'::currency),
  ('XCD'::currency),
  ('XAF'::currency),
  ('WST'::currency),
  ('VUV'::currency),
  ('VND'::currency),
  ('VEF'::currency),
  ('UZS'::currency),
  ('UYU'::currency),
  ('USD'::currency),
  ('UGX'::currency),
  ('UAH'::currency),
  ('TZS'::currency),
  ('TWD'::currency),
  ('TVD'::currency),
  ('TTD'::currency),
  ('TRY'::currency),
  ('TOP'::currency),
  ('TND'::currency),
  ('TMT'::currency),
  ('TJS'::currency),
  ('THB'::currency),
  ('SZL'::currency),
  ('SYP'::currency),
  ('SVC'::currency),
  ('STD'::currency),
  ('SRD'::currency),
  ('SPL'::currency),
  ('SOS'::currency),
  ('SLL'::currency),
  ('SHP'::currency),
  ('SGD'::currency),
  ('SEK'::currency),
  ('SDG'::currency),
  ('SCR'::currency),
  ('SBD'::currency),
  ('SAR'::currency),
  ('RWF'::currency),
  ('RUB'::currency),
  ('RSD'::currency),
  ('RON'::currency),
  ('QAR'::currency),
  ('PYG'::currency),
  ('PLN'::currency),
  ('PKR'::currency),
  ('PHP'::currency),
  ('PGK'::currency),
  ('PEN'::currency),
  ('PAB'::currency),
  ('OMR'::currency),
  ('NZD'::currency),
  ('NPR'::currency),
  ('NOK'::currency),
  ('NIO'::currency),
  ('NGN'::currency),
  ('NAD'::currency),
  ('MZN'::currency),
  ('MYR'::currency),
  ('MXN'::currency),
  ('MWK'::currency),
  ('MVR'::currency),
  ('MUR'::currency),
  ('MRO'::currency),
  ('MOP'::currency),
  ('MNT'::currency),
  ('MMK'::currency),
  ('MKD'::currency),
  ('MGA'::currency),
  ('MDL'::currency),
  ('MAD'::currency),
  ('LYD'::currency),
  ('LTL'::currency),
  ('LSL'::currency),
  ('LRD'::currency),
  ('LKR'::currency),
  ('LBP'::currency),
  ('LAK'::currency),
  ('KZT'::currency),
  ('KYD'::currency),
  ('KWD'::currency),
  ('KRW'::currency),
  ('KPW'::currency),
  ('KMF'::currency),
  ('KHR'::currency),
  ('KGS'::currency),
  ('KES'::currency),
  ('JPY'::currency),
  ('JOD'::currency),
  ('JMD'::currency),
  ('JEP'::currency),
  ('ISK'::currency),
  ('IRR'::currency),
  ('IQD'::currency),
  ('INR'::currency),
  ('IMP'::currency),
  ('ILS'::currency),
  ('IDR'::currency),
  ('HUF'::currency),
  ('HTG'::currency),
  ('HRK'::currency),
  ('HNL'::currency),
  ('HKD'::currency),
  ('GYD'::currency),
  ('GTQ'::currency),
  ('GNF'::currency),
  ('GMD'::currency),
  ('GIP'::currency),
  ('GHS'::currency),
  ('GGP'::currency),
  ('GEL'::currency),
  ('GBP'::currency),
  ('FKP'::currency),
  ('FJD'::currency),
  ('EUR'::currency),
  ('ETB'::currency),
  ('ERN'::currency),
  ('EGP'::currency),
  ('DZD'::currency),
  ('DOP'::currency),
  ('DKK'::currency),
  ('DJF'::currency),
  ('CZK'::currency),
  ('CVE'::currency),
  ('CUP'::currency),
  ('CUC'::currency),
  ('CRC'::currency),
  ('COP'::currency),
  ('CNY'::currency),
  ('CLP'::currency),
  ('CHF'::currency),
  ('CDF'::currency),
  ('CAD'::currency),
  ('BZD'::currency),
  ('BYR'::currency),
  ('BYN'::currency),
  ('BWP'::currency),
  ('BTN'::currency),
  ('BSD'::currency),
  ('BRL'::currency),
  ('BOB'::currency),
  ('BND'::currency),
  ('BMD'::currency),
  ('BIF'::currency),
  ('BHD'::currency),
  ('BGN'::currency),
  ('BDT'::currency),
  ('BBD'::currency),
  ('BAM'::currency),
  ('AZN'::currency),
  ('AWG'::currency),
  ('AUD'::currency),
  ('ARS'::currency),
  ('AOA'::currency),
  ('ANG'::currency),
  ('AMD'::currency),
  ('ALL'::currency),
  ('AFN'::currency),
  ('AED'::currency)
) t(c)
ORDER BY c;
  c  
-----
 AED
 AFN
 ALL
 AMD
 ANG
 AOA
 ARS
 AUD
 AWG
 AZN
 BAM
 BBD
 BDT
 BGN
 BHD
 BIF
 BMD
 BND
 BOB
 BRL
 BSD
 BTN
 BWP
 BYN
 BYR
 BZD
 CAD
 CDF
 CHF
 CLP
 CNY
 COP
 CRC
 CUC
 CUP
 CVE
 CZK
 DJF
 DKK
 DOP
 DZD
 EGP
 ERN
 ETB
 EUR
 FJD
 FKP
 GBP
 GEL
 GGP
 GHS
 GIP
 GMD
 GNF
 GTQ
 GYD
 HKD
 HNL
 HRK
 HTG
 HUF
 IDR
 ILS
 IMP
 INR
 IQD
 IRR
 ISK
 JEP
 JMD
 JOD
 JPY
 KES
 KGS
 KHR
 KMF
 KPW
 KRW
 KWD
 KYD
 KZT
 LAK
 LBP
 LKR
 LRD
 LSL
 LTL
 LYD
 MAD
 MDL
 MGA
 MKD
 MMK
 MNT
 MOP
 MRO
 MUR
 MVR
 MWK
 MXN
 MYR
 MZN
 NAD
 NGN
 NIO
 NOK
 NPR
 NZD
 OMR
 PAB
 PEN
 PGK
 PHP
 PKR
 PLN
 PYG
 QAR
 RON
 RSD
 RUB
 RWF
 SAR
 SBD
 SCR
 SDG
 SEK
 SGD
 SHP
 SLL
 SOS
 SPL
 SRD
 STD
 SVC
 SYP
 SZL
 THB
 TJS
 TMT
 TND
 TOP
 TRY
 TTD
 TVD
 TWD
 TZS
 UAH
 UGX
 USD
 UYU
 UZS
 VEF
 VND
 VUV
 WST
 XAF
 XCD
 XDR
 XOF
 XPF
 YER
 ZAR
 ZMW
 ZWD
(164 rows)

